python/nano/src/bigdl/nano/pytorch/strategies/ray/ray_distributed.py

```diff
@@ -127,25 +122,27 @@ def launch(self, function: Callable, *args: Any,
         futures = [
             strategy.workers[i].execute.remote(
                 self._wrapping_function,
-                (i, trainer, function, args, kwargs)
+                (i, trainer, strategy, function, args, kwargs)
             )
             for i in range(strategy.num_workers)
         ]
 
         results = ray.get(futures)  # type: ignore
 
+        if trainer is None:
+            return results[0]
+
         # Get the results, checkpoint path, and model weights from worker 0.
         results, best_path, state_dict = results[0]  # type: ignore
-        strategy._model.load_state_dict(state_dict)
+        strategy.model.load_state_dict(state_dict)
```

Comment:
is this a bug in the original code?

Reply:
not a bug, the `model` refer to `_model`, they are the same, change it for consistency